web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ExportCsvIT.java

```diff
@@ -744,13 +742,14 @@ private void assertContainerRecord(List<CSVRecord> csvList, ResourceType objType
             String path, int depth, boolean deleted, Integer numChildren, boolean described, String permissions,
                                        String embargoDate) {
         assertCsvRecord(csvList, objType, expectedPid, title, path, depth, deleted,
-                null, null, null, null, numChildren, described, permissions, embargoDate, "");
+                null, null, null, null, numChildren, described, permissions,
+                embargoDate, "", null, null);
     }
 
     private void assertCsvRecord(List<CSVRecord> csvList, ResourceType objType, PID expectedPid, String title,
             String path, int depth, boolean deleted, String mimetype, String checksum, Long fileSize,
             String accessSurrogate, Integer numChildren, boolean described, String permissions,
-            String embargoDate, String behavior) {
+            String embargoDate, String behavior, String parentWorkUrl, String parentWorkTitle) {
```

Comment:
Doesn't seem like parentWorkUrl or parentWorkTitle ever get used in this method.

Reply:
thanks for catching!